lib/modules/onboarding/rest_api.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Podlove\Modules\Onboarding;
+
+use Podlove\Modules\WordpressFileUpload\Wordpress_File_Upload;
+use WP_REST_Controller;
+
+class WP_REST_PodloveOnboarding_Controller extends WP_REST_Controller
+{
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        $this->namespace = 'podlove/v2';
+        $this->rest_base = 'onboarding';
+    }
+
+    /**
+     * Register the component routes.
+     */
+    public function register_routes()
+    {
+        register_rest_route($this->namespace, $this->rest_base, [
```

Comment:
Vielleicht lieber den Endpunkt benennen, falls wir später mehr als einen brauchen.

```suggestion
        register_rest_route($this->namespace, $this->rest_base . "/setup", [
```